js/main.js

```diff
@@ -12,13 +12,67 @@ const getRandomFloat = (min, max, digits) => {
     return null;
   }
   const multiplier = 10 ** digits;
-  // если количество знаков после запятой меньше чем у исходных значений,
-  // то случайное число будет получатся всегда вне заданного диапазона
-  if (!Number.isInteger(min * multiplier) || !Number.isInteger(max * multiplier)) {
-    return null;
-  }
   return Math.floor((Math.random() * (max - min) + min) * multiplier) / multiplier;
 };
 
-getRandomInt(1, 2);
-getRandomFloat(1.1, 1.2, 2);
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'
+];
+const FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+const TYPES = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+const TIMES = ['12:00', '13:00', '14:00'];
+
+const makeAuthor = () => {
+  const userNumber = getRandomInt(1, 10);
+  return {
+    avatar: `img/avatars/user${(userNumber === 10) ? userNumber : `0${userNumber}`}.png`
+  };
+};
+
+const makeLocation = () => ({
+  lat: getRandomFloat(35.65000, 35.70000, 5),
+  lng: getRandomFloat(139.70000, 139.80000, 5)
+});
+
+const makeOfferInfo = (location) => {
+  const offer = {};
+  offer.address = `${location.lat}, ${location.lng}`;
+  offer.price = getRandomInt(5, 100);
+  offer.type = TYPES[getRandomInt(0, TYPES.length)];
+  offer.rooms = getRandomInt(1, 50);
+  offer.guests = getRandomInt(1, 50);
+  offer.checkin = TIMES[getRandomInt(0, TIMES.length)];
+  offer.checkout = TIMES[getRandomInt(0, TIMES.length)];
+  offer.features = [];
+  for (const feature of FEATURES) {
+    if (getRandomInt(0, 2) === 1) {
```

Comment:
```suggestion
    if (Boolean(getRandomInt(0, 1))) {
```
крутое решение!
можно сделать чуть красивей)
0 - это false
1 - это true